app/controllers/experiments_controller.rb

```diff
@@ -96,7 +96,12 @@ def table
 
   def load_table
     active_modules = @experiment.my_modules.active
-    render json: Experiments::TableViewService.new(active_modules, current_user).call
+                                .page(params[:page] || 1)
```

Comment:
I think pagination must be done inside the service. We have `includes` logic inside the service.